### PR TITLE
GBFAsyncTaskGamePlayTagAddedOrRemoved cancellabe

### PR DESCRIPTION
**Pull request by ThibNach**
This change is [Reviewable](https://reviewable.io/reviews/TheEmidee/UEGameBaseFramework/236)

### CODE REVIEW

File: Source/GameBaseFramework/Public/GAS/BlueprintAsyncActions/GBFAsyncTaskGameplayTagAddedOrRemoved.h [hunk 1]
```diff
@@ -8,11 +8,11 @@
 
 #include "GBFAsyncTaskGameplayTagAddedOrRemoved.generated.h"
 
-DECLARE_DYNAMIC_MULTICAST_DELEGATE_OneParam( FSWOnGameplayTagChangedDelegate, bool, it_is_present );
+DECLARE_DYNAMIC_MULTICAST_DELEGATE_TwoParams( FSWOnGameplayTagChangedDelegate, bool, it_is_present, const UGBFAsyncTaskGameplayTagAddedOrRemoved *, listening_task );
```
Comment (Frickerson) [inline Source/GameBaseFramework/Public/GAS/BlueprintAsyncActions/GBFAsyncTaskGameplayTagAddedOrRemoved.h]: I would pass it as the first argument, for consistency with other tasks

Comment (ThibNach) [reply Source/GameBaseFramework/Public/GAS/BlueprintAsyncActions/GBFAsyncTaskGameplayTagAddedOrRemoved.h]: Done

**Review by Frickerson — DISMISSED** [review]

**Comment by TheEmidee** [comment]
Make the class inherit from `UCancellableAsyncAction` instead of passing it in the delegate.
While you're at it, you can change the parent class of all blueprint async actions in the folder

**Review by MaxDaniels — DISMISSED** [review]

**Review by Frickerson — APPROVED** [review]

**Review by TheEmidee — APPROVED** [review]
:lgtm:

> Reviewed 2 of 2 files at r2, 6 of 6 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/TheEmidee/UEGameBaseFramework/236#-:-NrdWGpf0ViXEbvbV6FK:bnfp4nl)* status: all files reviewed, 1 unresolved discussion (waiting on @VectorGamez)

**Pull request merged**